deployment/terraform/container_service.tf

```diff
@@ -0,0 +1,205 @@
+locals {
+  app_image = "${module.ecr_repository.repository_url}:${var.image_tag}"
+}
+
+#
+# Security Group Resources
+#
+resource "aws_security_group" "alb" {
+  vpc_id = "${module.vpc.id}"
+
+  tags {
+    Name        = "sgLoadBalancer"
+    Project     = "${var.project}"
+    Environment = "${var.environment}"
+  }
+}
+
+resource "aws_security_group" "ecs_service" {
```

Comment:
The name of this resource didn't immediately make sense to me. Based on its usage, I associate it with a specific service, not any ECS service. For example, if we had to add another service for background processing, that security group would need a service specific name.